packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -103,7 +107,14 @@ export default decorate([
                   pool.default_SR
                 ),
               })
-            )
+              const oldTemplates = filter(
+                templates,
+                template =>
+                  pool.$pool === template.$pool &&
+                  template.other['xo:resource:namespace'] === namespace
+              )
+              await Promise.all(oldTemplates.map(pureDeleteVm))
```

Comment:
Does this work? `map` will pass a second argument (the index of the loop) to `pureDeleteVm` and then `pureDeleteVm` will try to spread it. Or did I miss something?